bioblend/toolshed/__init__.py

```diff
@@ -19,6 +19,7 @@ def __init__(
         email: Optional[str] = None,
         password: Optional[str] = None,
         *,
+        token: Optional[str] = None,
```

Comment:
Actually I am not sure that https://github.com/galaxyproject/galaxy/pull/16977 adds support for authenticating via token also to the ToolShed API?

Reply:
Thanks for catching that.